src/lib/components/Feedback/index.js

```diff
@@ -6,18 +6,51 @@ import {BaseElement} from '../BaseElement';
 import {trackEvent} from '../../analytics';
 import './_styles.scss';
 
+/** @types {WebFeedback.Question[]} */
+const questions = [
+  {text: 'Was this page helpful?', label: 'helpful', name: 'Helpfulness'},
+  {
+    text: 'Did this page help you complete your goal(s)?',
+    label: 'effective',
+    name: 'Effectiveness',
+  },
+  {
+    text: 'Did this page have the information you needed?',
+    label: 'complete',
+    name: 'Completeness',
+  },
+  {
+    text: "Was this page's information accurate?",
+    label: 'accuracy',
+    name: 'Accuracy',
+  },
+  {text: 'Was this page easy to read?', label: 'readable', name: 'Readability'},
+];
+
+/** @types {WebFeedback.Question[]} */
+const conditionalQuestions = [
```

Comment:
Maybe rename to `additionalQuestions` and add a comment that these questions are optional and enabled by the `additional_feedback_questions` key?